File: Week4/Python/CCOOK.py
for _ in range(int(input())):
    A = input().split()
    a = A.count('1')
    if(a == 0):
        print('Beginner')
    elif(a == 1):
        print('Junior Developer')
    elif(a == 2):
        print('Middle Developer')
    elif(a == 3):
        print('Senior Developer')
    elif(a == 4):
        print('Hacker')
    else:
        print('Jeff Dean')

File: Week4/C++/CCOOK.cpp
#include <iostream>
using namespace std;

int main() {
	int n, a, solved=0;
	cin>>n;
	for(int i=0; i<n; i++)
	{
	    solved = 0;
	    for(int j=0; j<5; j++)
	        {
	            cin>>a;
	            if(a == 1)
	                solved++;
	        }
	        if(solved == 0)
	            cout<<"Beginner"<<endl;
	        else if(solved == 1)
	            cout<<"Junior Developer"<<endl;
	        else if(solved == 2)
	            cout<<"Middle Developer"<<endl;
	        else if(solved == 3)
	            cout<<"Senior Developer"<<endl;
	        else if(solved == 4)
	            cout<<"Hacker"<<endl;
	        else
	            cout<<"Jeff Dean"<<endl;
	}
	return 0;
}